src/Files.Uwp/UserControls/SidebarControl.xaml.cs

```diff
@@ -1072,7 +1073,7 @@ private void Border_PointerExited(object sender, PointerRoutedEventArgs e)
         {
             if (!dragging) // keep showing pressed event if currently resizing the sidebar
             {
-                Window.Current.CoreWindow.PointerCursor = new CoreCursor(CoreCursorType.Arrow, 0);
+                //App.Window.CoreWindow.PointerCursor = new CoreCursor(CoreCursorType.Arrow, 0); //WINUI3
```

Comment:
What's happening here?

Reply:
Can't use CoreWindow on WinUi3. Need to figure out replacement.

Reply:
What does this break? Can you add it to the progress issue?

Reply:
I think it causes this (already in the issue):
- [ ]  Grid splitters have the wrong cursor

Reply:
Thanks!

Reply:
Replacement may look somethink like: `this.ProtectedCursor = Microsoft.UI.Input.InputSystemCursor.Create(Microsoft.UI.Input.InputSystemCursorShape.SizeWestEast)`